Changelog.md

```diff
@@ -3,6 +3,7 @@
 - Adds `response` computed property to `Error` type, which yields a Response object if available.
 - Added URLEncodedInURL to ParameterEncoding.
 - Adds convenience `endpointByAdding` method.
+- **Breaking Change** Remove our own implementation of `ParameterEncoding` and make it a `public typealias` of `Alamofire.ParameterEncoding`.
```

Comment:
I suppose _if_ someone was overriding the `Endpoint` class or otherwise relying on access to the underlying Alamofire functionality... I just don't want to release a major version unnecessarily.